src/App.tsx

```diff
@@ -269,41 +294,49 @@ function App() {
         }
     );
 
-    usePageNumber(
-        page,
-        (page) => page === 5,
-        () => setIsPlaying(false)
-    );
-
-    // if they hit pause instead of clicking "Next", we still want to progress
+    // they have recorded a single value, changed the slider and pressed play
     usePageNumber(
         page,
         (page) =>
-            page === 4 && uniqMeasuredConcentrations.length > 0 && !isPlaying,
-        () => setPage(5)
+            page === 8 &&
```

Comment:
Would it make sense to replace `8` with a named constant? That way you can change it everywhere at once if a page gets pushed further back